GoogleMapsComponents/Maps/Extension/CircleList.cs

```diff
@@ -15,11 +15,15 @@ namespace GoogleMapsComponents.Maps.Extension
     ///
     /// All properties should be called With a Dictionary<string, {property type}> indicating for each Circle(related to that key) the corresponding related property value
     /// </summary>
-    class CircleList: IDisposable
+    class CircleList: BaseListableEntityList<Circle, CircleOptions>
```

Comment:
Wouldn't you want to make the class public?  
What would be the "correct way" to instantiate it? Is there a factory method I'm missing?

Reply:
I know. I just saw accidnelty that it not public. I was thinking to do in next release. I think it comes as VS dont make it public by default